components/eamxx/src/physics/share/physics_constants.hpp

```diff
@@ -127,6 +127,11 @@ struct P3_Constants
 {
   public:
   Scalar p3_autoconversion_prefactor = 1350.0;
+  Scalar p3_autoconversion_qc_exp     = 2.47;
```

Comment:
Random question: do we know why the struct `P3_Constants` is in `physics/share` rathe than `physics/p3`? I don't see it used anywhere else. Do we foresee other parts of eamxx to interact with it?

Reply:
I agree, I think we should pull it out of `physics/share`.  I fear that in the future we may see `physics/share/physics_constants.hpp` as a dumping ground for a bunch of different processes that may or may not be in use.

Reply:
Thanks Naser for volunteering to do it an a separate PR... :P

Reply:
(jk)